app/uk/gov/hmrc/economiccrimelevyregistration/navigation/RelevantApLengthPageNavigator.scala

```diff
@@ -18,16 +18,20 @@ package uk.gov.hmrc.economiccrimelevyregistration.navigation
 
 import play.api.mvc.Call
 import uk.gov.hmrc.economiccrimelevyregistration.controllers.routes
-import uk.gov.hmrc.economiccrimelevyregistration.models.Registration
+import uk.gov.hmrc.economiccrimelevyregistration.models.{NormalMode, Registration}
 
 class RelevantApLengthPageNavigator extends PageNavigator {
 
   override protected def navigateInNormalMode(registration: Registration): Call =
     registration.relevantApLength match {
-      case Some(_) => routes.UkRevenueController.onPageLoad()
+      case Some(_) => routes.UkRevenueController.onPageLoad(NormalMode)
       case _       => routes.JourneyRecoveryController.onPageLoad()
     }
 
-  override protected def navigateInCheckMode(registration: Registration): Call = ???
+  override protected def navigateInCheckMode(registration: Registration): Call =
+    registration.relevantApLength match {
+      case Some(_) => routes.CheckYourAnswersController.onPageLoad()
```

Comment:
Shouldn't we be recapturing revenue again rather than returning to the check your answers page?